visualization/dashboard/src/MLIDashboard/Controls/GlobalExplanationTab/GlobalExplanationTab.styles.ts

```diff
@@ -92,6 +96,28 @@ export const globalTabStyles: () => IProcessedStyleSet<IGlobalTabStyles> = () =>
         },
         missingParametersPlaceholder: [FabricStyles.missingParameterPlaceholder],
         missingParametersPlaceholderSpacer: [FabricStyles.missingParameterPlaceholderSpacer],
-        faintText: [FabricStyles.faintText]
+        faintText: [FabricStyles.faintText],
+        chartEditorButton: {
+            color: theme.semanticColors.buttonText,
+            backgroundColor: theme.semanticColors.buttonBackground,
+            borderWidth: "1px",
+            borderStyle: "solid",
+            borderColor: theme.semanticColors.buttonBorder,
+            position: "absolute",
+            right: "210px",
+            zIndex: 10
+        },
+        callout: {
+            width: "200px",
+            boxSizing:"border-box",
+            display: "flex",
+            flexDirection: "column",
+            padding: "10px 20px",
+            backgroundColor: theme.semanticColors.bodyBackground
+        },
+        boldText: {
+            fontWeight: "600",
```

Comment:
I found  "FontWeights" which I'm using and I'm wondering if you want to use `FontWeights.semibold` instead of harcoded values. 
https://github.com/microsoft/fluentui/blob/master/packages/styling/src/styles/fonts.ts

Reply:
nice!